src/middlewares/noAuth.middleware.js

```diff
@@ -0,0 +1,23 @@
+const { ApiError } = require('../exceptions/api.error');
+const jwtService = require('../services/jwt.service');
+
+const noAuthMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
+
+  if (authorization || token) {
+    throw ApiError.forbidden();
+  }
+
+  const userData = jwtService.verify(token);
```

Comment:
This line attempts to verify the token even if it might be undefined. Ensure that 'token' is defined before calling 'jwtService.verify(token)'.